runtime/compiler/trj9/env/J9VMEnv.hpp

```diff
@@ -111,6 +111,13 @@ class OMR_EXTENSIBLE VMEnv : public OMR::VMEnvConnector
    bool ensureOSRBufferSize(TR::Compilation *comp, uintptrj_t osrFrameSizeInBytes, uintptrj_t osrScratchBufferSizeInBytes, uintptrj_t osrStackFrameSizeInBytes);
    uintptrj_t thisThreadGetOSRReturnAddressOffset(TR::Compilation *comp);
 
+   uintptrj_t thisThreadGetGSIntermediateResultOffset(TR::Compilation *comp);
```

Comment:
These names would be better as things like getVMThreadGSIntermediateResultOffset in my opinion...